test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,10 @@
 const { joinNames } = require("../src");
-
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+    expect(joinNames([{name: 'Bart'}, {name: 'Lisa'}, {name: 'Maggie'}])).toEqual('Bart, Lisa & Maggie');
```

Comment:
Great stuff, it might be easier to read if you have your arguments saved as variables above the test case when they start getting long. Just for readability though.

Reply:
You could also save the person objects and then just have something like `const bart = { name: "Bart" }[bart, maggie]`